### ISSUE TITLE
Meetings?

### ISSUE THREAD

**Issue by silkentrance** [tusharvjoshi/NBDrupalDevel#51]
@tusharvjoshi How about getting together. As I was initially opting for taking over this project, and since you stepped in before I could even react, is there any chance of discussing future plans on how to bring this plugin out of its non existence? IRC/mail or other any other channels?

**Comment by tusharjoshi** [comment]
@silkentrance I was not aware of your wish to take over the project that time hence I jumped in
1.  I can still pass on the ownership to you, as you are associated with the project more than me
2. I will continue to contribute to this project with you, 

Let me know if you would like to go that way.

**Comment by tusharjoshi** [comment]
I have done certain changes to the branching policy in this repository and I would like it to remain that way for smooth releases.
- master branch - this branch shall be used only for releases.  That means once we have a tag on develop branch then only we should merge develop into master.  We do releases from master branch usually, if some additional commits needed like version bump etc., then we can have specific release branches for some time and then merge into master.  I have made master branch protected from force pushes in Github settings
- develop branch has been made the default branch.  This is the branch which gets cloned and should remain current HEAD for development and pull request merges.  Once we are convinced that the code is ready for release we merge develop into master

**Comment by silkentrance** [comment]
@tusharvjoshi no need for passing this on to me :smile:.

I'd like to be able to contact you outside of this platform, say IRC, to discuss further development. And I believe that you have plans, don't you? I for myself had plans, too, so we might consider bringing these plans together...

Since you are located in India, I believe, and I am located in Germany, we have a slight timezone problem. However, and since I am currently unemployed, I can shift time and adjust so that we can get in tough via real time channels such as IRC or whatever you prefer. Personally, I'd prefer IRC over Skype though. That way, communication is slower, but more understandable, me with my slow speaking tongue and bad english and you with your ever so fast speaking tongue :smile:

**Comment by silkentrance** [comment]
As for the branching and tagging policies, these are right along my alley.

**Comment by tusharjoshi** [comment]
I have created a Gitter community (as it looked like the easiest way to integrate with the Github project)  and have added you @silkentrance to that community to start with.  Advantage of this community is that it retains all the communication as an archive for others who join late in the party.

Hoping to have exchange of ideas and plans with you and many more interested contributors on that channel

**Comment by tusharjoshi** [comment]
I would like to mention one more change done to the original repository for reference purposes.  The binary file NetBeansDrupalDevelTool.nbm was committed to the repository from initial commit and was committed multiple times when updated increasing the size of the repository unnecessarily.

It was also redundant to keep binary files which can be generated through a build into the source control commits.  I have deleted all of those binary files using git filter-branch mechanism and effectively re-writing the history of the repository, and keeping all the commit comments as well as changes as they were.

We should not commit any binary artifact to the repository if that can be generated from source by build.  

As we have the Gitter chat room associated with the project now, hence closing this thread.  Let us continue our discussion on Gitter room.

**Comment by silkentrance** [comment]
@tusharvjoshi great. i will look into gitter now an then to see if you are online.
